mantis-runtime/src/main/java/io/mantisrx/runtime/executor/WorkerConsumerRemoteObservable.java

```diff
@@ -100,6 +100,7 @@ private void registerMetrics(Metrics metrics) {
     }
 
     @Override
-    public void stop() {}
+    public void close() throws IOException {
```

Comment:
why is there nothing to close for this executor?

Reply:
I'm guessing the Observable handles most of the behavior under the hood when the unsubscribe occurs. Or just depends on the whole JVM being blown away.